libraries/Microsoft.Bot.Builder.Ai.LUIS/LuisRecognizerMiddleware.cs

```diff
@@ -36,7 +38,7 @@ public class LuisRecognizerMiddleware : IMiddleware
         /// </summary>
         public const string Obfuscated = "****";
```

Comment:
The pattern of using this "Obfuscated" is really odd and just sets off all kind of weird red-flags. 

We removed it from the QnA Maker components

Reply:
Added bug for Emad.

Reply:
@cleemullins it's just a string constant to indicate that the key property includes sensitive data, could be null or empty if you prefer, what's the "pattern" and red flags just so I would understand? We aren't really "obfuscating" anything or editing any existing data structures. This "****" string goes to a brand new data structure. 

I could introduce a new data structure that's identical to the existing one and not include the key property, but I was trying to keep the diagnostic info mapped as much as possible to existing data structures to 1. enforce the schema and 2. avoid introducing a new data structure that the developer will have to map back to the existing schema. We need all the data in the existing data structure so there's no point in introducing a new one.